src/openforms/js/components/form/edit/tabs.js

```diff
@@ -64,6 +64,15 @@ const BASIC = {
             tooltip: 'This will be the initial value for this field, before user interaction.',
             input: true
         },
+        {
+            // This doesn't work as in native HTML forms. Marking a field as 'disabled' only makes it read-only in the
+            // UI, but the data is still sent to the backend.
+            type: 'checkbox',
+            label: 'Read only',
+            tooltip: 'Make this component read only',
+            key: 'disabled',
+            input: true
+        },
```

Comment:
As mentioned during standup - this is probably not applicable for all types of components and we should check them all to see if they make sense or not.

Reply:
For the file upload, password, signature, co-signing and select family member field I think it doesn't make much sense to have the read only option. I shall remove it for these fields.

For the select box, the checkboxes, radio buttons it could be useful to have it, but the styling is not good yet. 

The map I am not sure if it makes sense to have it, but the styling is not working yet for it :stuck_out_tongue:

Reply:
In the end I think I will just add it to the `TEXT_BASIC` tab